docs/source/user-guide/tutorials/build-pkgs.rst

```diff
@@ -54,19 +53,19 @@ Editing the meta.yaml file
       :widths: 10 90
 
       * - name
-        - pyinstrument
+        - click
       * - version
-        - "0.13.1" (or latest from
-          https://github.com/joerick/pyinstrument/releases)
+        - "6.7" (or latest from
+          https://github.com/pallets/click/releases)
       * - git_rev
-        - v0.13.1 (or latest from
-          https://github.com/joerick/pyinstrument/releases)
+        - 6.7 (or latest from
+          https://github.com/pallets/click/releases)
       * - git_url
-        - https://github.com/joerick/pyinstrument.git
+        - https://github.com/pallets/click.git
       * - imports
-        - pyinstrument
+        - click
       * - home
-        - https://github.com/joerick/pyinstrument
+        - https://github.com/pallets/click
       * - license
```

Comment:
Do I also remove the license entries here?

Reply:
sure